### DISCUSSION TITLE
Song Requests

### DISCUSSION THREAD

**Discussion by Masheen88** [carperbr/vocal-remover-frame-transformer#1] [General]
I made a spreadsheet to track song requests. Did not want to post this on YouTube directly and was also unsure of your email to add permissions to the sheet itself. If you want any formatting / etc.. let me know. Happy to make changes. 

https://docs.google.com/spreadsheets/d/1DVmm9C4S2ugg4wmUz2nJixNRRKoVeIEkEAOgILzeVPI/edit#gid=0

**Comment by carperbr** [comment]
Awesome, thanks! Will request access to that in a bit.

**Comment by carperbr** [comment]
Hey sorry, lost track of this. I think what I may end up doing is posting a pinned post on youtube, but I'm not totally sure. I may give that a shot first, but I definitely appreciate you doing this. May end up using it still if a pinned post on youtube doesn't work well

**Reply by Masheen88** [reply]
Hey, no problem. Let me know what works best. Happy to help out.
